com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Deferred/RaytracingDeferred.compute

```diff
@@ -78,6 +78,7 @@ void RAYTRACING_DEFERRED(uint3 dispatchThreadId : SV_DispatchThreadID, uint2 gro
     PositionInputs posInput;
     posInput.positionWS = GetCameraRelativePositionWS(intersectionPositionWS);
     posInput.positionSS = currentCoord;
+    posInput.positionNDC = sourcePosInput.positionNDC;
```

Comment:
Raytracing shaders don't initialize posInput.positionNDC so it creates a compilation error. (But do we need to sample planar reflections when raytracing ?)

Reply:
yes, we still need it